### ISSUE TITLE
Add CMSes with GraphQL APIs

### ISSUE THREAD

**Issue by sedubois** [postlight/awesome-cms#64]
I would love to see more CMSes in this list which expose a GraphQL API or, even better, allow to manage a remote GraphQL endpoint.

I think at least Mozaik CMS should be added, haven't tried yet but seems quite promising. https://www.mozaik.io/

It might also be worth adding Prisma which converts any DB into a GraphQL endpoint with an admin dashboard, even if it's not a full-fledged CMS.

And worth mentioning GraphiQL itself, not a CMS but it does already offer content-editing capabilities without needing to directly fiddle with a DB.

I guess the line between admin panels and CMS can be a bit blurry as well, might be worth stating what is eligible or not in this list, or just accept everything with a label "admin" or "CMS".

**Comment by philwareham** [comment]
This is getting into the territory of https://github.com/chentsulin/awesome-graphql isn't it (and yes, I think GraphQL is great?) This awesome CMS list should probably have a limited scope to tools that can provide the complete process from content creation to end user delivery IMHO.

**Comment by sedubois** [comment]
Yeah whatever it covers it might help to mention it in the readme. And in all cases at least Mozaik could be added. Maybe add that Contentful has a read-only GraphQL adapter. And I’m sure there must be other CMSes with some GraphQL abilities, which is why I’d love to see this list completed with more things...
